pkgs/development/rocm-modules/5/composable_kernel/default.nix

```diff
@@ -15,7 +15,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "composable_kernel";
-  version = "unstable-2023-01-16";
+  version = "unstable-2023-09-29";
```

Comment:
I think that they have started releasing stable versions for composable_kernel: https://github.com/ROCmSoftwarePlatform/composable_kernel/releases

Reply:
Yes, I'm planning on switching to it when I get to `composable_kernel`.
`miopengemm` may or may not be removed, since it's incredibly out-of-date.

Reply:
Changed in recent commit.